src/controls/utils.ts

```diff
@@ -7,3 +7,7 @@ export function decimalToPercentage(value?: number) {
 export function percentageStringToDecimal(value: string | undefined) {
   return Number(value) / 100
 }
+
+export const coerceToInteger = (value: number) => Math.round(value)
+
+export const coerceToNumber = (value: number) => Number(value)
```

Comment:
What do you think about deleting these altogether and solving the coercion problem in the anyOf or array controls?

Reply:
It sounds like a better approach. I think we would also need to solve it in `ObjectControl`. Can we defer that for another PR?